core/lib/state/src/catchup.rs

```diff
@@ -49,7 +141,10 @@ impl AsyncCatchupTask {
     /// Propagates RocksDB and Postgres errors.
     pub async fn run(self, stop_receiver: watch::Receiver<bool>) -> anyhow::Result<()> {
         let started_at = Instant::now();
-        tracing::debug!("Catching up RocksDB asynchronously");
+        tracing::debug!(
+            "Catching up RocksDB asynchronously to L1 batch #{:?}",
+            self.to_l1_batch_number
+        );
```

Comment:
nit: This is kind of opinionated but wouldn't this be cleaner (structured logs and all that good stuff)?

```suggestion
        tracing::debug!(
            target_l1_batch = ?self.to_l1_batch_number,
            "Catching up RocksDB asynchronously",
        );
```

Reply:
Implemented it in a slightly different way, via `tracing::instrument` spans.